src/services/baseService.test.ts

```diff
@@ -84,48 +85,60 @@ describe("Base Service", () => {
   it("returns region and stage based on CLI options", () => {
     const cliOptions = {
       stage: "prod",
-      region: "eastus2"
+      region: "eastus2",
     };
     const mockService = new MockService(sls, cliOptions);
 
     expect(mockService.getRegion()).toEqual(cliOptions.region);
     expect(mockService.getStage()).toEqual(cliOptions.stage);
   });
 
-  it("Sets default region and stage values if not defined", () => {
-    const mockService = new MockService(sls);
-
-    expect(mockService).not.toBeNull();
-    expect(sls.service.provider.region).toEqual("westus");
-    expect(sls.service.provider.stage).toEqual("dev");
-  });
-
-  it("returns region and stage based on CLI options", () => {
+  it("use the resource group name specified in CLI", () => {
```

Comment:
Consider renaming the test description to be inclusive of region and stage values in addition to the resource group.

Reply:
there's already another test for region and stage.  I just thought separating the test make more sense because we don't always specific the region, stage, and resourceGroup flag together.  Thoughts?

Reply:
OK - from the diff it looks like you are replacing the existing text.